README.md

```diff
@@ -157,6 +157,7 @@ If you have a website, you can query against the usage statistics of your site:
     ```js
    browserslist('> 5% in my stats', { stats: 'path/to/the/stats.json' });
     ```
+5. Import statistic from `browserslist-stats.json` in project folder
```

Comment:
It is not a separated step.  It is just part of 4 step. Also feel free to rewrite section to make it more clear.